data_structures/src/chain.rs

```diff
@@ -3856,6 +3853,13 @@ mod tests {
         signature::sign,
     };
 
+    use crate::{
+        superblock::{mining_build_superblock, ARSIdentities},
+        transaction::{CommitTransactionBody, RevealTransactionBody, VTTransactionBody},
+    };
+
+    use super::*;
```

Comment:
Is this possible?
```suggestion
    use crate::{
        Self::*,
        superblock::{mining_build_superblock, ARSIdentities},
        transaction::{CommitTransactionBody, RevealTransactionBody, VTTransactionBody},
    };
```

Reply:
I don't think so, `crate::` means `/` and `super::` means `../`, combined they form `/../` which doesn't make sense.

Reply:
If `super::*` means `every sibling of this module` and this is a 2-nd level module, how's that not equivalent to `crate::*`?

Reply:
You mean it's redundant? I can remove the `use super::*` and see if it compiles

Reply:
Ah no, this is inside `mod tests`, so `super::*` refers to `crate::chain::*`. That's a common pattern used in tests so I'm leaving it as `super::*`.

Reply:
Oh, now I understand. Resolved!